package-lock.json

```diff
@@ -1886,7 +1886,7 @@
       "from": "github:snapshot-labs/snapshot-spaces#master"
     },
     "@snapshot-labs/snapshot.js": {
-      "version": "github:snapshot-labs/snapshot.js#914d6c58e354720907a50f438754da0ae313afe8",
+      "version": "github:snapshot-labs/snapshot.js#2d0d815d5176a3143b8a779b4a3f7c6c2ae41dbf",
```

Comment:
Why this? This is an old commit

Reply:
I'm not sure I messed up the branch somewhere when I started work on this.